scripts/base/init-bare.zeek

```diff
@@ -4278,7 +4278,9 @@ export {
 	## Name and flags for a single channel requested by the client.
 	type RDP::ClientChannelDef: record {
 		## A unique name for the channel
-		name:           string;
+		name:           string;	
+		## Channel Def raw options as count
+		options:	int;
```

Comment:
Looks like that should be a `count` instead of `int` for the "options" field. The spec says it is unsigned (and also you're internally using `val_mgr->GetCount` to assign it anyway).